### DISCUSSION TITLE
Insert a video，gitbook can play directly in current page not download

### DISCUSSION THREAD

**Discussion by 123sijia** [GitbookIO/community#802] [Feature Requests]
I want to insert a video into the document. I hope that after publishing, it can be played directly in the document or played in a rendering player, instead of only being able to download or open a youtube link.

**Comment by stephen-ashby** [comment]
Hey there! 

How are you adding the video to your page? If you paste a video link into GitBook and hit `Enter` it should automatically embed that video so that readers can play it by clicking it. This works for most video URLs, including YouTube and direct links to mp4 files. 

If you drag and drop a video file into the GitBook editor it will default to being a download link. If you want to embed that video directly, you can click the link to open it in a new tab, then copy the direct link to the video, paste it into an empty line in GitBook and hit `Enter`. That should embed it.

Take a look [at our docs](https://docs.gitbook.com/content-editor/blocks/embed-a-url) to find out more. Hope that helps — let us know how you get on!
